File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int>mp(nums.begin(),nums.end());
     int ans=0;
        for(int num:mp){

            if(mp.find(num-1)==mp.end()){
              int curr=num;
              int seq=1;
                while(mp.find(curr+1)!=mp.end()){
                    curr++;
                    seq++;
                }
                ans=max(ans,seq);

            }
        }
        return ans;
        

    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        
        int size=nums.length;

        int[] ans=new int[2*size];

        for(int i=0;i<2*size;i++){
            ans[i]=nums[i%size];
        }
        return ans;
        
    }
}